.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Bug Report
+<!--
+    If you are not submitting a bug report, ignore this template and see
+      http://Chapel-Issues-Best-Practices.com  (TODO)
+  -->
+### Summary of Problem
+
+
+### Steps to Reproduce
+
+**What behavior do you observe when encountering this issue?**
+
+**Please provide one of the following:**
+
+- Reduced program demonstrating issue *(preferred)*:
+- Original program demonstrating issue:
+- Description of how to recreate the issue *(last resort)*:
```

Comment:
I wonder whether we should remove this option.  It's intended more for private bugs where people can't share source code, but if we're handling such cases via a mailing list (since this is, by definition, public), not opening that possibility here seems preferable.  And if someone's really sensitive about what they're sharing, they'll figure out a way to open the issue while ignoring this prompt.